<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelListDlg</class>
 <widget class="QDialog" name="ChannelListDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/16x16/actions/oxygen/16x16/actions/format-list-unordered.png</normaloff>:/16x16/actions/oxygen/16x16/actions/format-list-unordered.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="searchPatternLabel">
       <property name="text">
        <string>Search Pattern:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="channelNameLineEdit"/>
     </item>
     <item>
      <widget class="ClickableLabel" name="advancedModeLabel">
       <property name="toolTip">
        <string>Toggle between simple and advanced mode.
Advanced mode allows one to pass search strings to the IRC Server.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/22x22/actions/oxygen/22x22/actions/edit-clear-locationbar-rtl.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchChannelsButton">
       <property name="text">
        <string>Show Channels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string>Errors Occurred:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="errorTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;SUPER EVIL CATASTROPHIC ERROR!!11&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="channelListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>channelNameLineEdit</tabstop>
  <tabstop>searchChannelsButton</tabstop>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>errorTextEdit</tabstop>
  <tabstop>channelListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
